public/scss/cart.scss

```diff
@@ -0,0 +1,194 @@
+@import "breakpoint.scss";
+
+.cart-container{
+    background-color: #E0E0E0;
+    max-width: 1142px;
+    max-width: 1142px;
+    margin: 10px auto;
+    
+}
+
+.shopping-cart {
+    display: flex;
+    background-color: black;
+    width: 100%;
+    height: 3.5em;
+    justify-content: space-between;
+    padding: 20px;
+
+    .myCart{
+        position: relative;
+        color: white;
+
+        #cart-value{
```

Comment:
If we are putting rules on ID then why nested rule? You can directly target this element instead of putting inside shopping-cart

Reply:
Done